compiler.rst

```diff
@@ -7,23 +7,14 @@ Design of CPython's Compiler
 Abstract
 --------
 
-Historically (through 2.4), compilation from source code to bytecode
-involved two steps:
+In CPython, the compilation from source code to bytecode involves three steps:
```

Comment:
> Note 2.

I've reviewed the rst markup as well as run the file through [restructuredtext_lint](https://pypi.python.org/pypi/restructuredtext_lint). Looks good.